CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Modify the prover to match the paper [#831]
 - Modify the verifier to match the paper [#831]
 - Rename some variables to match the paper [#831]
+- Modify the verifier to be faster [#834]
```

Comment:
I would say "make use of parallelization to improve the verifier time" (make a bit more explicit what these modifications entail)

Reply:
done